Sources/ClientRuntime/Retries/DefaultRetryStrategy/DefaultRetryToken.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.TimeInterval
+
+/// A token that is used to track retry of operations.
+///
+/// The retry token contains all the state relevant to one request that is needed to manage retry
+/// until the request succeeds or fails after zero or more retries.
+public class DefaultRetryToken: RetryToken {
```

Comment:
This is the retry token used by the default retry strategy.

Per standard architecture, it holds all state related to one request's retries.